floodviz/tests/test_map_utils.py

```diff
@@ -68,20 +105,24 @@ def setUp(self):
                              'alt_acy_va': '.01', 'alt_datum_cd': 'NGVD29'}]
 
     def test_empty_list(self):
-        self.assertEqual(site_dict([], self.prefix), [])
+        with requests_mock.Mocker() as m:
```

Comment:
This isn't really necessary but harmless. If you had a more general url matching it might make more sense (e.g. any url with self.mock_prefix. I'm okay with leaving it.